app/controllers/admin/reviews_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Admin::ReviewsController < AdminController
+  def destroy
+    @review = Review.find(params[:id])
+    @bookstore = @review.bookstore
+    @review.destroy!
```

Comment:
Again--what about an error here? What if the page is stale and the review cannot be found because another admin deleted it already?